pydy/forces.py

```diff
@@ -0,0 +1,35 @@
+from sympy import Symbol
+
+__all__ = ['Force', 'GravitationalForce']
+
+
+class Force(object):
+    """
+    It provides an concept of force, although actual force is added by system.
+
+    Parameters
+    ---------
+    point: sympy.physics.vector.Point
+        Point of application of force.
+    direction: sympy.physics.vector.Vector
+        Direction of application of force.
+    magnitude: (optional)
```

Comment:
I just wanted to make things clear for a user so that he by mistake doesn't include the magnitude of direction vector iteself in the force but keeping it simple is also fine.